src/index.html

```diff
@@ -6,9 +6,39 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+          crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <div class="header__wrap">
+          <a href="#" class="logo"><img src="./images/logo.png" alt="moyo"></a>
```

Comment:
Alt must clearly describe the image.
```suggestion
          <a href="#" class="logo"><img src="./images/logo.png" alt="moyo logo"></a>
```